storybook/src/components/TextArea/TextArea.docs.mdx

```diff
@@ -31,3 +31,21 @@ import README from "../../../../packages/css/src/components/text-area/README.md?
 ## Disabled
 
 <Canvas of={TextAreaStories.Disabled} />
+
+### In a Field
+
+Use a Field component to group a Text Area with a Label, description and / or an error message.
+
+If you don’t need the description, just take out the description Paragraph and the `aria-describedby` from the Text Input.
+
+Check [the Field component documentation](/docs/components-forms-field--docs) for more information on configuring it.
```

Comment:
```suggestion
Check [Field](/docs/components-forms-field--docs) for more information on configuring it.
```

Or ‘Consult [the Field docs]` if you really want the ‘documentation’ piece in?

Reply:
Just 'Field' is a bit terse imo. How about 'Check the Field docs ...'?

Reply:
Alright. I’ve applied this to all references like this one.